readthedocs/api/v2/serializers.py

```diff
@@ -133,11 +136,49 @@ class Meta:
         exclude = []
 
 
+class BuildCommandUISerializer(BuildCommandSerializer):
```

Comment:
I would name these `Read`/`ReadOnly`, since it isn't necessarily just the UI that use these, but we use them when listing/retrieving a record

```suggestion
class BuildCommandReadSerializer(BuildCommandSerializer):
```